website/docs/integrations/intellij.mdx

```diff
@@ -0,0 +1,148 @@
+---
+id: intellij
+title: JetBrains/IntelliJ Plugin - Manage your feature flags from JetBrains/IntelliJ IDEs
+description: ConfigCat IntelliJ IDEA Plugin. This is a step-by-step guide on how to use the ConfigCat JetBrains/IntelliJ  Plugin to manage feature flags in your IDEs.
+---
+
+// TODO jetbrains mp link
+
+<a href="https://marketplace.visualstudio.com/items?itemName=ConfigCat.configcat-feature-flags" target="_blank">ConfigCat's JetBrains/IntelliJ IDE plugin</a> to manage feature flags from IDEs.
+
+Connect your ConfigCat Product and Config to your JetBrains/IntelliJ IDE. Find your Feature Flag's usages in your code easily.
+
+This guide will help you install the JetBrains/IntelliJ IDE Plugin and familiarize you with its usage.
+
+## Feature overview
+
+- View your Feature Flags within the IDE.
+- Create Feature Flags within the IDE.
+- Open a Feature Flag on the ConfigCat Dashboard.
+- Find Feature Flag usages in your code.
+- Copy a Feature Flag's key to the clipboard.
+- View your Products & Configs.
+- Create Configs within the IDE.
+- Open a Config on ConfigCat Dashboard.
+
+<img src="/docs/assets/intellij/usage1.gif" className="zoomable" alt="Usage of ConfigCat Feature Flags & Settings Tab" />
+
+## Install plugin
+
+### Install within the IDE from Marketplace
+
+1. Open the `File` > `Settings/Preferences` menu.
```

Comment:
```suggestion
1. In your IDE, open the `File` > `Settings/Preferences` menu.
```

Minden bekezdés elején érdemes elmondani, hogy "HOL" kell csinálni az akciókat. Sokszor egyértelmű a userek nagy részének, de néha nem, és nekik tudunk segíteni.

Ugyanígy érdemes elmondani a "HOL"-t, akkor is amikor másik toolra kell váltani a usernek.